src/components/SearchResultList/SearchResultList.tsx

```diff
@@ -87,7 +90,13 @@ const SearchResultList = () => {
           <tbody data-testid="search-table__body">
             {searchResults.map((result) => {
               return (
-                <tr key={result.id} className="table__body-row">
+                <tr
+                  onClick={() => {
+                    router.push(`/searchResults/${result.id}`);
```

Comment:
```suggestion
                    router.push(`/search-results/${result.id}`);
```


Anyway, I'm starting to wonder why are we using every word like `result` or `search result` or `tag`, but, `keyword` 👨

```suggestion
                    router.push(`/keywords/${keyword.id}`);
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/82/commits/14b3e809724c325626c187058af635fd02b3984c

Reply:
You already using the `search result` in so many places, so I think it will be better if you continue with `search-results`. Only change it to be `keywords` here but still using `search result` on other places will make it consistent. 👀

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/82/commits/6adae3c702d3ba0e498629267f601397a3c1024d